Makefile

```diff
@@ -0,0 +1,35 @@
+GH_USERNAME := thoth-station
+BRANCH := main
+
+all: # NOOP
+
+prepare: # Render dev kustomization.yaml interpolating GH_USERNAME and BRANCH so you can deploy from fork
+	BRANCH=$(BRANCH) GH_USERNAME=$(GH_USERNAME) envsubst < manifests/overlays/dev/kustomization.yaml.tmpl > manifests/overlays/dev/kustomization.yaml
+
+deploy: prepare # Applies manifests on a cluster
+	kustomize build manifests/overlays/dev | oc apply -f -
+
+undeploy: prepare # Deletes manifests from a cluster
+	kustomize build manifests/overlays/dev | oc delete -f -
+
+run: # Creates and follows execution of 1 positive and 1 negative cases of validation
+	tkn pipeline start byon-import-jupyterhub-image \
+		--showlog \
+		-w name=data,volumeClaimTemplateFile=manifests/overlay/dev/pvc.yaml \
```

Comment:
The PVC template is missing from the PR

Reply:
ah, wait, it's not missing - there's a path typo (`overlay` vs `overlays`)

Reply:
I do not think it's missing.. https://github.com/open-services-group/byon/blob/e2d3b50d3999fdb68de38a468e2be9029b5c9960/manifests/overlays/dev/pvc.yaml

Actually. I think it's a typo, it should be `overlays` not `overlay` here, lol. :facepalm: